server/models/userProfile.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose')
+
+const reqString = {
+  type: 'string',
+  required: true
+}
+
+const noReqString = {
+  type: 'string',
+  required: false
+}
+
+const date = {
+  type: Date,
+  default: null
+}
+
+const userProfile = new mongoose.Schema(
+  {
+    firstName: reqString,
```

Comment:
Even though its a personalized choice, I would prefer to this format as its much more readable and easily modifiable
```
firstName: {
  type: 'string',
  required: false}
```